charts/prometheus-redis-exporter/templates/servicemonitor.yaml

```
@@ -1,5 +1,5 @@
 {{- if and ( .Capabilities.APIVersions.Has "monitoring.coreos.com/v1" ) ( .Values.serviceMonitor.enabled ) }}
-apiVersion: monitoring.coreos.com/v1
+apiVersion: {{ .Values.serviceMonitor.apiVersion }}
```

Backport for users who did not set apiVersion.

```suggestion
apiVersion: {{ .Values.serviceMonitor.apiVersion | default "monitoring.coreos.com/v1" }}
```